packages/rancher-vsphere-csi/charts/templates/vsphere-csi-controller-deployment.yaml

```diff
@@ -25,10 +25,24 @@ spec:
         node-role.kubernetes.io/controlplane: "true"
       {{- end }}
       tolerations:
+        # Rancher specific change: These tolerations are intentionally different from upstream to avoid lessening the scope to only NoSchedule with a specific key
+        # - key: node-role.kubernetes.io/master
+        #   operator: Exists
+        #   effect: NoSchedule
```

Comment:
Why are we adding commented out tolerations ? Does this example come from somewhere else

Reply:
The commented out toleration is a toleration that was set upstream for this update, but based on a conversation with @aiyengar2, we decided to reject that change from upstream and keep the tolerations as is because of [this](https://github.com/rancher/charts/pull/1135#discussion_r621552035). I kept the toleration commented out so that we know it is a Rancher specific change next time we update it because it is difficult to distinguish between what we and upstream removed/added.